shell/src/main/java/alluxio/shell/command/TailCommand.java

```diff
@@ -85,11 +98,32 @@ void runCommand(AlluxioURI path, CommandLine cl) throws IOException {
 
   @Override
   public String getUsage() {
-    return "tail <path>";
+    return "tail -C <number of bytes> <path>";
   }
 
   @Override
   public String getDescription() {
-    return "Prints the file's last 1KB of contents to the console.";
+    return "Prints the file's last user specified number of bytes (by default, 1KB) of contents "
+        + "to the console.";
+  }
+
+  @Override
+  protected Options getOptions() {
+    Option bytesOption =
+        Option.builder("C")
+              .required(false)
+              .numberOfArgs(1)
+              .desc("user specified option")
+              .build();
+    return new Options().addOption(bytesOption);
+  }
+
+  @Override
+  public boolean validateArgs(String... args) {
```

Comment:
@xgong I did some testing and found that `alluxio fs tail -c 1024 /path` works fine without overriding validateArgs. `AbstractShellCommand#parseAndValidateArgs` calls `validateArgs(cmd.getArgs())`, and `cmd.getArgs()` returns only the arguments that were not parsed as options.